core/build.gradle

```diff
@@ -11,13 +11,8 @@ dependencies {
 
     compileOnly "com.discord4j:discord-json-encoding:$discordJsonVersion"
 
-    if (JavaVersion.current().isJava8()) {
-        compileOnly "com.google.code.findbugs:annotations:3.0.1"
-        testCompileOnly "com.google.code.findbugs:annotations:3.0.1"
-    } else {
-        compileOnly "javax.annotation:javax.annotation-api:1.3.2"
-        testCompileOnly "javax.annotation:javax.annotation-api:1.3.2"
-    }
+    compileOnly "com.google.code.findbugs:jsr305:3.0.1"
+    testCompileOnly "com.google.code.findbugs:jsr305:3.0.1"
```

Comment:
Based in https://github.com/Discord4J/Discord4J/pull/1220/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7 this can me move to the main build.gradle for handle all modules

Reply:
Done